workers/data_refinery_workers/processors/management/commands/run_tximport.py

```diff
@@ -67,6 +67,6 @@ def handle(self, *args, **options):
         pjofa.original_file = original_file
         pjofa.save()
 
-        final_context = tximport.tximport(job.id)
+        tximport.tximport(job.id)
```

Comment:
It seems like we'd probably want that final_context...

Reply:
for what? doesn't look like is needed

Reply:
I think most of instances of `final_context` were relics from Rich functional testing his code. Occasionally they'd be used in unit tests as well, but not always.